core/src/main/java/nu/yona/server/analysis/service/IntervalActivityDTO.java

```diff
@@ -0,0 +1,55 @@
+package nu.yona.server.analysis.service;
+
+import java.util.Date;
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import nu.yona.server.analysis.entities.IntervalActivity;
+import nu.yona.server.goals.service.GoalDTO;
+
+@JsonRootName("intervalActivity")
+public class IntervalActivityDTO
+{
+	private GoalDTO goal;
+	private Date startTime;
+	private Date endTime;
+	private Set<ActivityDTO> activities;
```

Comment:
What is this for? an ``IntervalActivity`` doesn't have activities inside, right?